src/main/java/teammates/ui/controller/InstructorFeedbackResultsPageData.java

```diff
@@ -47,6 +49,9 @@
     
     private static final int RESPONDENTS_LIMIT_FOR_AUTOLOADING = 150;
 
+    private static int sectionId;
```

Comment:
Opps, this probably shouldn't be a field. I meant the `Pattern` (which you have now done) in that comment a few days ago, so this sectionId can remain a local variable where it is used